modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -15258,6 +15258,7 @@ service=Service
 service-access-policy-entry-default-asset-entry-title=Public Access to the Asset Entry API
 service-access-policy-entry-default-asset-tag-title=Public Access to the Asset Tag API
 service-access-policy-entry-default-calendar-title=Public Access to the Calendar Search API
+service-access-policy-entry-default-captcha-entry-title=Public Access to Captcha API
```

Comment:
service-access-policy-entry-default-captcha-title=Public Access to Captcha API

Reply:
Fixed it. Thanks